test/catalog/catalog_test.cpp

```diff
@@ -365,8 +408,23 @@ TEST_F(CatalogTests, LayoutCatalogTest) {
   txn = txn_manager.BeginTransaction();
   auto default_layout =
       catalog->CreateDefaultLayout(database_oid, table_oid, default_map, txn);
-  txn_manager.CommitTransaction(txn);
   EXPECT_NE(nullptr, default_layout);
+  txn_manager.CommitTransaction(txn);
+
+  // Check the changed default layout
+  auto default_layout_oid = default_layout->GetOid();
+  EXPECT_EQ(default_layout_oid, table->GetDefaultLayout()->GetOid());
+  EXPECT_FALSE(default_layout->IsColumnStore());
+  EXPECT_FALSE(default_layout->IsRowStore());
```

Comment:
Can we test if its type is `HYBRID` directly?

Reply:
No, there was not a function to access the layout type directly. These two `EXPECT_FALSE` tests match HYBRID actually.

Reply:
I see. How about just adding a function like `IsHybridStore()` to make it more clear?